core-ui/src/components/Predefined/Details/Namespace/NamespaceWorkloads/NamespaceWorkloads.js

```diff
@@ -85,7 +85,7 @@ export function NamespaceWorkloads({ namespace }) {
       <LayoutPanel.Header>
         <Icon
           size="m"
-          className="fd-has-margin-right-small"
+          className="fd-margin-end--smmall"
```

Comment:
I could drop a suggestion here, but there are more double 'm's, it'd be better to replace all them in Code.